### PR TITLE
eth-rpc-errors

### PR DESCRIPTION
**Pull request by estebanmino**
`eth-json-rpc-errors` -> `eth-rpc-errors` update

### CODE REVIEW

File: src/user/PreferencesController.ts [hunk 1]
```diff
@@ -182,10 +182,9 @@ export class PreferencesController extends BaseController<BaseConfig, Preference
 		addresses = addresses.map((address: string) => toChecksumAddress(address));
 		const oldIdentities = this.state.identities;
 		const identities = addresses.reduce((ids: { [address: string]: ContactEntry }, address, index) => {
-			ids[address] = {
+			ids[address] = oldIdentities[address] || {
```
Comment (Gudahtt) [inline src/user/PreferencesController.ts:185]: This change seems functionally equivalent, as long as the `name` and `address` properties are always set on pre-existing identities (which I think is true?). Not sure what it has to do with the other changes in this PR though.

Comment (estebanmino) [reply src/user/PreferencesController.ts:185]: I had issues with typescript errors, without those changes I couldn't build it. About your question, yes they're always defined.

image

Comment (Gudahtt) [reply src/user/PreferencesController.ts:185]: Ah I see - that makes sense. Thanks!

Comment (whymarrh) [reply src/user/PreferencesController.ts:185]: This seems similar to #243—@estebanmino what version of TS are you using?

Comment (estebanmino) [reply src/user/PreferencesController.ts:185]: 3.8.3

Comment (estebanmino) [reply src/user/PreferencesController.ts:185]: Is the same issue I had, but I'm on 3.8.3

**Pull request merged**